client/my-sites/plugins/main.jsx

```diff
@@ -55,7 +55,7 @@ export default React.createClass( {
 	mixins: [ URLSearch, PluginNotices ],
 
 	getInitialState() {
-		return this.getPluginsState( this.props );
+		return Object.assign( this.getPluginsState( this.props ), { bulkManagement: [] } );
```

Comment:
I don't know enough about how they are used, but if it's all derived data that starts fresh on each load then I think you have a valid point @johnHackworth 

I like to ask, "should this persist across a page reload?" and if so then it belongs in the store but likely not otherwise